### ISSUE TITLE
Pretraining Dataset size

### ISSUE THREAD

**Issue by cuuupid** [little-misfit/BooW-VTON#5]
Hi @little-misfit , very interesting paper!

I noticed this bit that really stood out to me regarding synthetic data:

> Specifically, in our designed training paradigm, we first conduct the conventional try-on pipeline in simple in-shop scenarios through a powerful pretrained try-on model (i.e. IDM-VTON). Then, considering the source person image as the ground truth of the try-on, the previously generated try-on image and source cloth image can form a standard training pair to achieve wild try-on in a mask-free approach.

Existing datasets like VITON and DressCode aren't that large in size but if I understand this approach correctly you could generate a dataset multiples larger via the above approach.

I'm wondering how many training pairs are in the dataset you created this way? And, did you try any ablation studies to observe the effect of synthetic dataset size on FID/KID?

**Comment by little-misfit** [comment]
Thank you for your recognition! The number of synthetic virtual pairs we generated is consistent with the original dataset (VITON-HD or DressCode), meaning each clothing item corresponds to only one virtual pair. 
However, in the in-the-wild augmentation, we enhanced the data through augmentation by randomly adding foregrounds and backgrounds during each training step (with minimal computational cost). In theory, the images are different in each training step. Additionally, we only conducted ablation experiments on the in-the-wild data augmentation, not on the specific number of virtual pairs. However, that's a great idea, and we will consider adding it in future work! ~o(*￣▽￣*)ブ
